pipelines/nextflow/modules/files/collect_files_db.nf

```diff
@@ -45,11 +45,10 @@ process MANIFEST {
     time '5min'
 
     input:
-        path collect_dir
-        val db
+        tuple val(db), path(collect_dir)
     
     output:
-        path collect_dir
+        tuple val(db), path(collect_dir, includeInputs: true)
```

Comment:
I would have expected the output to include the input if you are using an input variable, but good to know `includeInputs` is needed :nerd_face: